packages/backend-modules/auth/express/auth.js

```diff
@@ -56,8 +63,8 @@ exports.configure = ({
       cookie: {
         domain,
         maxAge: maxAge,
-        sameSite: !dev && 'none',
-        secure: !dev,
+        sameSite: cookieOptions.sameSite,
+        secure: cookieOptions.secure,
```

Comment:
Wouldn't it make more sense to spread `cookieOptions` and always do domain via there?

Reply:
Probably yes, however I don't know what the motivation was behind passing all arguments to auth.js its self. I'd like Pae's opinion on that.